### ISSUE TITLE
data acquisition scripts should be added (?)

### ISSUE THREAD

**Issue by klaragerlei** [MattNolanLab/in_vivo_ephys_openephys#3]
- bonsai script
- open ephys acquisition protocol
- arduino script for synchronizing position and ephys data + opto stimulation

Maybe we should make another folder 'data_acquisition' and add them. We could also have a different repo like for the vr scripts, what do you think @stennant ?

**Comment by stennant** [comment]
I think this is a great idea @klaragerlei :) The VR acquisition protocols could be added too under a prefix VR_

**Comment by klaragerlei** [comment]
The VR scripts appear to have their own repo:
https://github.com/MattNolanLab/VR_blender_scripts_opto

**Comment by stennant** [comment]
Yeah I think it might be a good idea to keep them in one place. This repo could be tidied and yes, given a clear wiki and perhaps contain all the versions of the VR codes then keep the main up-to-date ones in the new repo. Not sure

**Comment by klaragerlei** [comment]
we can also merge them and that way we keep the history of the VR code which is good in my opinion.

https://stackoverflow.com/questions/1425892/how-do-you-merge-two-git-repositories
